FC-0068 docs: How-to cleaning by edunext #9 (#714)
openedx/docs.openedx.org: source/educators/how-tos/advanced_features/add_cohorts.rst

#. Click **Add Cohort**.

#. Enter a name for the cohort.

   .. note::
    Learners can see the name of the cohort they are assigned to. The message
    "This post is visible only to {cohort name}" appears with each post in
    discussion topics that are divided by cohort. See :ref:`Read the Group
    Indicator in Posts`.

#. Specify whether learners are automatically or manually assigned to this
   cohort.

#. Optionally, select **Select a Content Group** to associate the cohort with a
   :ref:`content group<About Content Groups>`. For information about creating
   cohort-specific course content by associating cohorts with content groups,
   see :ref:`Cohorted Courseware Overview`.

#. Select **Save**.

Continue implementing your cohort strategy by creating additional cohorts as
applicable, and specifying the assignment method for each cohort.

.. note:: By the time your course starts, you must have at least one cohort in
   your course that is automatically assigned. If you have not created at
   least one automated assignment cohort in the course by the time that the
   first learner accesses your course content or a divided discussion topic,

>    Open edX creates a default cohort to which learners are automatically assigned.

Reviewer: ```suggestion
   the platform creates a default cohort in your course to which learners are automatically assigned.
```
